src/lib/clients/ansible.js

```diff
@@ -20,6 +20,14 @@ class Ansible {
   }
 
   async sync() {
+    const util = require('util');
+    const exec = util.promisify(require('child_process').exec);
+
+    async function ls() {
```

Comment:
No please, use `this._cmd` for calling commands, the same as the rest of the module does.